main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: cat
+//  action 1: the cat fights
+cat.fight();
+//  action 2: the cat scratches
+cat.scratch();
+//  action 3: the cat meows
+cat.meow();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bowling ball
+//  action 1: the bowling ball rolls down the lane
+bowlingBall.rollsDownLane();
+//  action 2: the bowling ball knocks pins down
+bowlingBall.knocksPinsDown();
+//  action 3: the bowling ball goes in the gutter
+bowlingBall.goesInGutter();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: guitar player
+//  action 1: the guitar player makes a chord
+guitarPlayer.makesChord();
+//  action 2: the guitar player plays a note
+guitarPlayer.playsNote();
+//  action 3: the guitar player plays a song
+guitarPlayer.playsSong();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.